### PR TITLE
feat(rust): add commands to add members and get credentials

### PR DESCRIPTION
**Pull request by twittner**

### CODE REVIEW

File: implementations/rust/ockam/ockam/src/credential.rs
Comment (SanjoDeundiak) [inline implementations/rust/ockam/ockam/src/credential.rs]: I don't think we want base32 in our codebase. The decision was to use hex everywhere for the time being

Comment (SanjoDeundiak) [reply implementations/rust/ockam/ockam/src/credential.rs]: @twittner is data_encoding better than hex crate we already use?

**Review by adrianbenavides — COMMENTED** [review]
Looks good to me! :+1:

File: implementations/rust/ockam/ockam/src/credential.rs
Comment (twittner) [reply implementations/rust/ockam/ockam/src/credential.rs]: I was hoping we could agree to use a more compact encoding, but for the time being I removed data-encoding and use hex as requested.

Comment (SanjoDeundiak) [reply implementations/rust/ockam/ockam/src/credential.rs]: @twittner I think we're open to suggestions, but to be clear: making a change and expecting reviewer to notice it, is not the way to open a discussion

File: implementations/rust/ockam/ockam_command/src/project/mod.rs
Comment (SanjoDeundiak) [inline implementations/rust/ockam/ockam_command/src/project/mod.rs:80]: Can these please look just like every other match statement for command?

File: implementations/rust/ockam/ockam_command/src/project/get_credential.rs
Comment (SanjoDeundiak) [inline implementations/rust/ockam/ockam_command/src/project/get_credential.rs:39]: Does it work without secure channel?

File: implementations/rust/ockam/ockam_command/src/project/add_member.rs
Comment (SanjoDeundiak) [inline implementations/rust/ockam/ockam_command/src/project/add_member.rs:37]: Does it work without secure channel?

**Review by SanjoDeundiak — DISMISSED** [review]

File: implementations/rust/ockam/ockam_command/src/project/get_credential.rs
Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/project/get_credential.rs:39]: Katharina is working on the `project` resolution helper for multiaddr. Once we have that, when the multiaddr includes a project (`/project/default/service/forwarder_to_blue/service/api`), it will retrieve it and create the secure channel. So, we should probably wait until we have that in place, or merge it as it is and add that later.

File: implementations/rust/ockam/ockam_command/src/project/mod.rs
Comment (mrinalwadhwa) [reply implementations/rust/ockam/ockam_command/src/project/mod.rs:80]: The conciseness of this style is nice but let's keep it consistent since we have a lot of new contributors exploring this part of the code.

@twittner let's file a `good first issue` to change to the more concise style across all commands.

File: implementations/rust/ockam/ockam_command/src/project/get_credential.rs
Comment (mrinalwadhwa) [reply implementations/rust/ockam/ockam_command/src/project/get_credential.rs:39]: If the PR is ready, I suggest merging it without secure channels and add secure channels as part of the project change @spacekookie has been working on. 

This way we don't have to spend time rebasing and making the PR ready for merge again

**Comment by mrinalwadhwa** [comment]
Landing this since I am working on various commands and it would be better if we don't spend time on monday resolving conflicts. Let's tackle any suggested changes in follow on PRs

**Review by mrinalwadhwa — DISMISSED** [review]

**Review by mrinalwadhwa — APPROVED** [review]

**Pull request merged**
